src/features/organizations/l10n/messageIds.ts

```diff
@@ -35,6 +35,9 @@ export default makeMessages('feat.organizations', {
     loading: m('Loading...'),
     noLocation: m('No physical location'),
     partOfProject: m<{ projectLink: ReactElement }>('Part of {projectLink}'),
+    participatingInfo: m<{ participatingCount: number }>(
+      '{participatingCount} {participatingCount, plural, =1 {person is} other {persons are}} participating'
+    ),
```

Comment:
This works fine for most languages (and the only really important thing here is that it works for English) but in general we use a different form, like this:

```suggestion
    participatingInfo: m<{ participatingCount: number }>(
      '{participatingCount, plural, =1 {# person is} other {# persons are}} participating'
    ),
```